vision/google/cloud/vision/_gax.py

```diff
@@ -66,7 +68,12 @@ def annotate(self, images=None, requests_pb=None):
             requests = requests_pb
 
         annotator_client = self._annotator_client
-        responses = annotator_client.batch_annotate_images(requests).responses
+        try:
+            api_result = annotator_client.batch_annotate_images(requests)
+            responses = api_result.responses
+        except RetryError as exception:
```

Comment:
I guess what I had in mind was not necessarily catching this one instance of a `RetryError`. I either want to do that _all_ the time, or I want to clearly document to people what they should do if they get one. As a user, I am not sure how I would have been expected to know that `.cause.exception()` gets me the original exception.